backend/server.js

```diff
@@ -82,11 +83,65 @@ const FBReq = (req, cb) => {
 };
 
 // This should come from a table.
-const groups = { iterate: 410797219090898, 
-		 ArmTechCongress: 214940895208239, 
-		 socialbridgeapp: 629600800545917, 
+const groups = { iterate: 410797219090898,
+		 ArmTechCongress: 214940895208239,
+		 socialbridgeapp: 629600800545917,
 		 MICArmenia:195461300492991 };
 
+const endpoint = 'https://translate.yandex.net/api/v1.5/tr.json';
+const yandexTranslatorApiKey = process.env.ITERATE_YANDEX_TRANSLATOR_API;
+let jsonRequest = async (url, params, cb) => {
+  let handler = (err, res) => {
+    if (err)
+      return cb(err);
+    let obj;
+    try {
+      obj = JSON.parse(res.body);
+    } catch(e) {
+      cb(e);
+    }
+    cb(null, obj);
+  };
+  if (params.get === true)
+    await request_prom.get(url, handler);
+  else
+    await request_prom.post(url, params, handler);
+};
+// let detect = function detect(text, opts, cb) {
+//   jsonRequest(endpoint + '/detect', {
+//         form: {
+//           text: text,
+//           key: opts.key,
+//           format: opts.format
+//         }
+//      }, cb);
+// };
+let translate = async (text, opts, cb) => {
+  await jsonRequest(endpoint + '/translate', {
+      form: {
+        text: text,
+        key: opts.key,
+        format: opts.format,
+        lang: opts.from ? opts.from + '-' + opts.to : opts.to
+      }
+  }, cb);
+};
+let translateAll = async (textToTranslate) => {
+  let title_en;
+  let title_ru;
+  let title_hy;
+  await translate(textToTranslate, {to: 'en', format: 'text', key: yandexTranslatorApiKey}, (err, res) => {
+    title_en = res.text[0];
+  });
+  await translate(textToTranslate, {to: 'ru', format: 'text', key: yandexTranslatorApiKey}, (err, res) => {
+    title_ru = res.text[0];
+  });
+  await translate(textToTranslate, {to: 'hy', format: 'text', key: yandexTranslatorApiKey}, (err, res) => {
+    title_hy = res.text[0];
+  });
+  let title = title_en + ' / ' + title_hy + ' / ' + title_ru;
```

Comment:
(implementation not required for this PR) I've been wondering if this is too long, from a usability perspective. This could be replaced w/ an object `{en: 'en_str', hy: 'hy_str', ru: 'ru_str'}` ref: http://formatjs.io/guides/basic-i18n/.